include/libunwind-aarch64.h

```diff
@@ -34,6 +34,11 @@ extern "C" {
 #include <inttypes.h>
 #include <stddef.h>
 #include <ucontext.h>
+#include "compiler.h"
```

Comment:
I believe this file is a public header whereas `compiler.h` is not. `compiler.h` currently have way too many defs that are not "namespaced" (including the fille name itself) to be changed to a public header. Since `ALIGNED` seems to be the only one needed anywhere in this patch it might just be better to copy part of that logic here with a different name, or even. If this kind of thing happens more then it'll worth creating a new public header for this...

Reply:
I pushed a patch to resolve this.